### ISSUE TITLE
broken init-DiffAlg.m2

### ISSUE THREAD

**Issue by dimpase** [Macaulay2/M2#1023]
`init-DiffAlg.m2` creates and uses global variable `i`, where it's not intended to be global. 
This has been discovered on an "incomplete" build, where documentation was not ready yet.
As a result, we got
```
... 
i1: help
...
Macaulay2/packages/Macaulay2Doc.m2:26:1:(2):[15]: error: global symbols inadvertently defined by package Macaulay2Doc: 'i'
/home/dima/.Macaulay2/init-DiffAlg.m2:18:24-18:25: here is the first use of 'i'
stdio:1:1:(3): error: after eval: --backtrace update-- 
```

**Comment by DanGrayson** [comment]
Yes, something should be done about that.

**Comment by DanGrayson** [comment]
@DiffAlg

**Comment by DiffAlg** [comment]
Ok, we are on it...

**Comment by mahrud** [comment]
Fixed in 691ba7ed3c
